packages/helpers/src/locales/fr.yml

```diff
@@ -0,0 +1,11 @@
+internal:
+  error-encountered: "Une erreur inconnue s'est produite : {0}"
+
+admin:
+  user-not-found: Utilisateur introuvable.
+  user-unchanged: Les données utilisateur n'ont pas été modifiées.
+  user-updated: Les données utilisateur ont été changées.
+  channel-not-found: Chanel introuvable.
+  channel-unchanged: Les données du chanel n'ont pas été modifiées.
+  channel-updated: Les données du chanel ont été changées.
+  not-in-group: Vous n'êtes pas dans le contexte du groupe, tapez -c pour spécifier le chanel cible.
```

Comment:
Then I am not sure if the `guild` can be translated as `groupe`, or should it be `compagnie` or something?

Reply:
Depending on the context it can be all 3 : guilde, groupe, companie. It depends of the game (wow will use guilde and ffxiv will use companie).

Reply:
`not-in-group: Vous n'êtes pas dans le contexte du groupe, tapez -c pour spécifier le canal cible.`

Reply:
🤣lol, just realized that chanel is that famous brand of a french company. How can I make this stupid mistake!

Reply:
the "guild" here refers to "a group of channels and users". Like in discord, it is called a "server" that has many channels and users.

Reply:
Groupe is fine then 👍

Reply:
Voilà ! Merci beaucoup, mooondark !